### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

**Comment by chzesa** [comment]
Thank you for your contribution. While it'd be perfectly ok to merge as is, I'm afraid I'd forgotten to use internationalization for a context menu entry. I've now also added the options page strings to messages.json.
If you can add translations for these new strings I'll hold off on merging this.

**Comment by Vistaus** [comment]
Done! 🙂

File: src/_locales/nl/messages.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	, "sidebarContextMenuMoveToExistingWindowPlural": {
+		"message": "Venster $WINDOW_ID$ ($TAB_COUNT$ tabbladen - actief: $ACTIVE_TAB_TITLE$)"
+		, "placeholders": {
+			"window_id": { "content" : "$1" }
+			, "tab_count": { "content" : "$2" }
+			, "active_tab_title": { "content": "$3" }
+		}
+		, "description": "Wordt getoond als het venster meer dan 1 tabblad heeft"
+	}
+}
```
Comment (chzesa) [inline src/_locales/nl/messages.json]: The closing brace on line 104 causes a parsing error.

**Review by chzesa — CHANGES_REQUESTED** [review]

File: src/_locales/nl/messages.json (hunk 1 above)
Comment (Vistaus) [reply src/_locales/nl/messages.json]: Good catch! I just fixed it. I blame GNOME's Meld for this error 😒

**Pull request merged**